pkg/buildinfo/buildinfo_unit.go

```diff
@@ -1,33 +0,0 @@
-//go:build unit
-// +build unit
-
-/*
-Copyright 2023 The Dapr Authors
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package buildinfo
-
-import (
-	"strings"
-)
-
-// Comma-separated list of features to enable in unit tests
-const unitTestFeatures = "Resiliency"
```

Comment:
Sorry, we really need this file to be able to hardcode certain features :(

Reply:
How about adding the same arguments of `ldflags` to `go test`?

I made the changes in the PR a few days ago in the Makefile.

Reply:
In general, I think it's better to keep the arguments of `go build` and `go test` as consistent as possible, so it tests the same behaviors as it ships.